hspec-core/src/Test/Hspec/Core/Extension/Tree.hs

```diff
@@ -0,0 +1,19 @@
+module Test.Hspec.Core.Extension.Tree (
+  SpecTree
+, mapItems
+, filterItems
+) where
+
+import           Prelude ()
+import           Test.Hspec.Core.Compat
+
+import           Test.Hspec.Core.Spec hiding (SpecTree)
+import qualified Test.Hspec.Core.Spec as Core
+
+type SpecTree = Core.SpecTree ()
+
+mapItems :: (Item () -> Item ()) -> [SpecTree] -> [SpecTree]
+mapItems = bimapForest id
```

Comment:
Wouldn't `fmap . fmap` be better?

Reply:
I'm somewhat undecided, yes, you are right that's true, but would this make the code any more clean?

Reply:
I think most people know what `fmap . fmap` does especially for the type `[SpecTree]`, but `bimapForest` is a specialized function that would require reference.

Reply:
I guess `map . fmap` works for me.